force-app/main/default/lwc/accountModelSettings/accountModelSettings.js

```diff
@@ -1,9 +1,20 @@
-import { LightningElement, api, track } from "lwc";
+import { LightningElement, api, track, wire } from "lwc";
+
+import getAccountModelSettingsViewModel from "@salesforce/apex/AccountModelSettingsController.getAccountModelSettingsViewModel";
+
 import stgAccountModelSettingsTitle from "@salesforce/label/c.stgAccountModelSettingsTitle";
+import stgAccModelTitle from "@salesforce/label/c.stgAccModelTitle";
+import stgHelpAccountModel from "@salesforce/label/c.stgHelpAccountModel";
+import stgAdminAccountRecordType from "@salesforce/label/c.stgAdminAccountRecordType";
+import stgHelpAdminRecType from "@salesforce/label/c.stgHelpAdminRecType";
+import stgAccountRecordTypeSupportsHHAddress from "@salesforce/label/c.stgAccountRecordTypeSupportsHHAddress";
+import stgHelpHouseholdRecType from "@salesforce/label/c.stgHelpHouseholdRecType";
+import AfflMappingsDescription from "@salesforce/label/c.AfflMappingsDescription";
```

Comment:
We don't use this label anywhere
```suggestion
import AfflMappingsDescription from "@salesforce/label/c.AfflMappingsDescription";
```